raiden_contracts/data/gas.json

```diff
@@ -1,16 +1,16 @@
 {
     "EndpointRegistry.registerEndpoint": 48408,
     "MonitoringService.claimReward": 42679,
-    "MonitoringService.monitor": 203858,
-    "OneToN.claim": 90914,
+    "MonitoringService.monitor": 201313,
+    "OneToN.claim": 90897,
```

Comment:
I remember that there was an issue with raiden when these were lowered, but when switching to the new contracts completely this shouldn't be a problem.

Reply:
When the last problem happened, the numbers lived in a global `raiden_contracts/constants.py`.  That set of numbers affected all versions. Now the situation should be better. `gas.json` lives in `data_x.y.z` so the numbers are local to a particular version.

But I have to try the new way to see if any problem happens.